modules/ROOT/pages/genai-integrations.adoc

```diff
@@ -142,22 +142,33 @@ CREATE (:Page { index: index, text: resource, vector: vector })-[:OF]->(book)
 .Generate embeddings for many text properties
 ====
 
-If you want to generate embeddings for the text content of all nodes with the label `Tweet`, you can use `CALL ... IN TRANSACTIONS` to split the work up into batches and issue one API request per batch.
+If you want to generate embeddings for the text content of all nodes with the label `Tweet`, you can
+divide the nodes up into batches, and issue one API request per batch.
 
-Assuming nodes with the `Tweet` label have a `text` property, you can generate vector embeddings for each one and write them to the `embedding` property on each one in batches of 1000, for example:
+Assuming nodes with the `Tweet` label have a `text` property, you can generate vector embeddings
+for each one and write them to their `embedding` property in batches of a thousand at a time, for example.
```

Comment:
```suggestion
for each one and write them to their `embedding` property in batches of a thousand at a time.
```

Reply:
The "for example" referred to the batch size of a thousand, which is pretty arbitrary. Would you prefer some other phrasing to emphasize that?

Reply:
Ah, I didn't get it. I thought it was a leftover.

Reply:
Perhaps we can add it after batches, like: `in batches, for example, of a thousand at a time.`

Reply:
In that case maybe `in batches of, for example, a thousand at a time.`? 🙂